src/data.cc

```diff
@@ -14,11 +14,14 @@
 #include "data/libfm_parser.h"
 #include "data/csv_parser.h"
 
+// To help C Preprocessor with processing c++ templated types
+#define COMMA ,
```

Comment:
COMMA is a too typical name, since it is helper do __DMLC_COMMA

Reply:
Done